docs/user-guide/src/quick-start.md

```diff
@@ -426,7 +426,9 @@ resources:
 components:
 - https://github.com/metal3-io/baremetal-operator/ironic-deployment/components/basic-auth?ref=v0.5.1
 - https://github.com/metal3-io/baremetal-operator/ironic-deployment/components/tls?ref=v0.5.1
-
+images:
+  - name: quay.io/metal3-io/ironic
```

Comment:
```suggestion
- name: quay.io/metal3-io/ironic
```
Indent this the k8s way, ie. no indentation with lists.

Reply:
done